telemetry/statsd/src/test/scala/io/buoyant/telemetry/statsd/MetricTest.scala

```diff
@@ -0,0 +1,98 @@
+package io.buoyant.telemetry.statsd
+
+import com.timgroup.statsd.StatsDClient
+import org.scalatest._
+
+class MetricTest extends FunSuite {
+
+  class MockStatsDClient extends StatsDClient {
+
+    var lastName = ""
+    var lastValue = ""
+
+    def count(x$1: String, x$2: Long, x$3: Double, x$4: String*): Unit = {
```

Comment:
yup, i grabbed all of it from an error output. i'll move it into its own file and use real var names for the relevant methods.